src/css/import.css

```diff
@@ -19,3 +19,12 @@
 #wb-import-fileInFiles .import-status .import-status-message .text-danger small {
     white-space: pre-wrap;
 }
+
+.impex-popover {
+    max-width: 470px;
+}
+
+.impex-popover pre {
+    white-space: pre-wrap;
+    word-wrap: break-word;
```

Comment:
I don't like this approach because this will actually force the browser to break lines exactly on the spot where the containing box ends which is undesireable. Better use `workd-wrap: normal` in order to achieve normal hyphenation wherever its possible and just then a force break.

Reply:
I removed the css style for here and also removed the `break-word` class from the `<td>` tag. This will set `word-wrap: normal` as it is the set by the `popover` class.

Reply:
great